Bottom: 1a0c9490f6ead8df787531d7396dd55ed2d4bc59
Top:    480438477d5794bf391cc29574a9d58fd3613593
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-20 09:51:00 -0500

Update configuration for running behind reverse proxy



---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index d23f6eee7..656c05a3e 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -47,7 +47,7 @@ container_env:
   BUILDBOT_SMTP_USER: 'buildbot'
   BUILDBOT_SMTP_PASS: '{{ buildbot_email_password }}'
   BUILDBOT_WEB_PORT: '{{buildbot_http_port}}'
-  BUILDBOT_WEB_URL: 'http://{{ buildbot_hostname }}/'
+  BUILDBOT_WEB_URL: 'https://{{ buildbot_hostname }}/'
   BUILDBOT_WORKER_PORT: '{{buildbot_worker_port}}'
   BUILDBOT_WORKER_NTEL: 'worker-ntel'
   BUILDBOT_WORKER_NTEL_PASS: '{{ buildbot_worker_ntel_password }}'
